src/Sql/Sql/AdvancedThreatProtection/Cmdlet/EnableAzureSqlManagedInstanceAdvancedDataSecurity.cs

```diff
@@ -59,7 +60,7 @@ protected override ManagedInstanceAdvancedDataSecurityPolicyModel PersistChanges
 
             if (DoNotConfigureVulnerabilityAssessment)
             {
-                ModelAdapter.SetManagedInstanceAdvancedDataSecurity(model);
+                ModelAdapter.SetManagedInstanceAdvancedDataSecurity(model, DefaultContext.Environment.GetEndpoint(AzureEnvironment.Endpoint.StorageEndpointSuffix));
```

Comment:
Why not add `DefaultContext.Environment.GetEndpoint(AzureEnvironment.Endpoint.StorageEndpointSuffix)`  to SqlAdvancedDataSecurityAdapter.cs and then you can remove the extra parameter from `SetManagedInstanceAdvancedDataSecurity`.

Like this:
`SqlThreatDetectionAdapter.SetManagedInstanceThreatDetectionPolicy(threatDetectionPolicy, DefaultContext.Environment.GetEndpoint(AzureEnvironment.Endpoint.StorageEndpointSuffix));`

Then you just have to change one file :)

Reply:
This comment is by me, I accidentally used my other account :)